### DISCUSSION TITLE
Proposal to go quickly to 10%, 20%, 30%.. of pages by pressing 1, 2, 3... key.

### DISCUSSION THREAD

**Discussion by kawabata** [sumatrapdfreader/sumatrapdf#3343] [Ideas]
Hi.
I read a lot of books with SumatraPDF.
I sometimes want to go through the whole pages of the book.
It would be conveninent if I press 1, 2, 3,... 9 button and go to 10%, 20%, 30%, ... 90% of pages of the book.  (for 0 and 100%, "Home" and "End" keys are appropriate.)
(This is a feature of "zippla" (https://sites.google.com/site/riostoolbox/zippla) book reader and I frequently use it.)  How is this idea?

**Comment by GitHubRulesOK** [comment]
If you press the up arrow while click on sidebar 1/10th of the way down then widows will jump to that 10% position
its a native system navigation ability without any need to replace
